modules/common/services/desktop.nix

```diff
@@ -30,7 +30,7 @@ in {
       profiles.graphics.compositor = "labwc";
       graphics = {
         launchers = let
-          hostEntry = filter (x: x.name == "ghaf-host-debug") config.ghaf.networking.hosts.entries;
+          hostEntry = filter (x: x.name == "ghaf-host" + lib.optionalString config.ghaf.profiles.debug.enable "-debug") config.ghaf.networking.hosts.entries;
```

Comment:
`ghaf-host` will not be reachable from guivm in release at all, any commands must be proxied over admin-vm which requires a 'private' network with the host.

Reply:
This PR aimed to solve the build error only and quickly. We can fix functionally in GIVC, however if it is needed I can make it with GIVC too.